_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "index"
```

Comment:
No no, this should be the title of your series. It's used to generate the title in multiple places, including the title bar and on the pages itself.

Seriously man, you need to pay attention to detail! :D Just look at the other files and remember, detail *matters.*